custom_components/venta/humidifier.py

```diff
@@ -59,9 +59,9 @@ async def async_setup_entry(
         return
 
     entity: VentaBaseHumidifierEntity
-    if coordinator.api.device.api_version == VentaApiVersion.V0:
+    if coordinator.api.device.api_version.value == VentaApiVersion.V0.value:
```

Comment:
This is unnecessary, since python handles it by default: https://docs.python.org/3/howto/enum.html#comparisons

Reply:
I was confused by this too

It didn't work for me somehow
I too would expect, that the original code would work, but it was always using V3 for me, despite the coordinator.api.device.api_version being VentaApiVersion.V0

Reply:
Which python version are you using?

Reply:
The Version the Home Assistant Docker Container is using

Python 3.12.1

Reply:
Could you check what's the type of `coordinator.api.device.api_version` in the runtime on your side?

Reply:
Also a lot of other places is using enum comparison so there must be something else going on.

Reply:
I think it might be just because something strange happened in between migrations. Could you remove add the device again via HA?